nav2_smac_planner/src/a_star.cpp

```diff
@@ -273,6 +335,9 @@ bool AStarAlgorithm<NodeT>::createPath(
     return false;
   }
 
+  NodeVector coarse_list, fine_list;
```

Comment:
```suggestion
  NodeVector coarse_check_goals, fine_check_goals;
```

This is a bit more descriptive of their intent / use

Reply:
done. https://github.com/ros-navigation/navigation2/pull/4127/commits/33e296f240030f00da596d3236b8faa28fe9fa0d